python/py-roman/Portfile

```
@@ -18,17 +18,26 @@ long_description    {*}${description}.
 
 homepage            https://github.com/zopefoundation/roman
 
-checksums           rmd160  793b600babe15e5cc811f9eb9fe2fe931378a180 \
-                    sha256  2c46ac8db827d34e4fa9ccc0577e7f0b0d84f16ffe112351bd4f1ec2eb12d73f \
-                    size    7577
+checksums           rmd160  f35c7998cc7e4dcddfb349992f7209096be855a3 \
+                    sha256  6caab7ba51b83c46b5bf8839e5d782769a6b08fdea8abdfa62a4197c041fd513 \
+                    size    9380
 
 python.versions     27 35 36 37 38 39 310 311
```

According to the Python Package Index, roman requires Python >= 3.7.

```suggestion
python.versions     37 38 39 310 311
```

---

As @dgilman mentioned, py-docutils still requires those subports. Maybe it's worth pinning py-roman at 3.3 for those older pythons.